app/assets/stylesheets/modern-business.scss

```diff
@@ -193,8 +193,8 @@ footer {
   vertical-align: middle;
 }
 
-#cart-td {
-  vertical-align: middle;
+#cart-btn {
+  text-decoration: none !important
```

Comment:
!important should not be used<br>Declaration should be terminated by a semicolon